tracing-subscriber/src/lib.rs

```diff
@@ -240,7 +240,7 @@ feature! {
     #![all(feature = "registry", feature = "std")]
     pub use registry::Registry;
 
-    ///
+    #[allow(missing_docs)]
```

Comment:
This function is used quite a lot, maybe we should add a small comment – something like "Creates a default Registry"?

Reply:
Fair, done.